Mathlib/Algebra/Group/Invertible/Basic.lean

```diff
@@ -30,7 +30,7 @@ theorem isUnit_of_invertible [Monoid α] (a : α) [Invertible a] : IsUnit a :=
   ⟨unitOfInvertible a, rfl⟩
 
 /-- Units are invertible in their associated monoid. -/
-def Units.invertible [Monoid α] (u : αˣ) :
+instance Units.invertible [Monoid α] (u : αˣ) :
```

Comment:
Looks like this is slowing down things enough that one `infer_instance` times out. Let me bench this and possibly revert it depending on the result.